opentech/apply/funds/models/submissions.py

```diff
@@ -396,6 +398,12 @@ class ApplicationSubmission(
         related_name='submissions',
         blank=True,
     )
+    activities = GenericRelation(
+        Activity,
```

Comment:
If you wrap this in a string it gives the reader a little more insight into which Activity model you're talking about since you need the app name if it's not in the same app.  Slight improvement: it also removes the requirement to order model definitions based on dependent models using them.

I won't gate the review on it though, your choice.